pkg/metaserver/util/util.go

```diff
@@ -1,25 +1,60 @@
 package util
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
 	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apiserver/pkg/storage"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/klog/v2"
 
 	beehiveModel "github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/client"
 )
 
 const (
 	EmptyString = ""
+	CrdGroup    = "apiextensions.k8s.io"
+	CrdVersion  = "v1beta1"
 )
 
+var CRDMapper *meta.DefaultRESTMapper
+
+// SyncCrdResource() is used to trigger the synchronization of CustomResourceDefinition resource
+func SyncCrdResource() error {
+	if _, err := client.GetCRDClient().ApiextensionsV1beta1().CustomResourceDefinitions().Watch(context.TODO(), metav1.ListOptions{}); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
I understand that it should be to synchronize the resources in the storage to the cache. I don't know why the watch method is used here?

Reply:
@zc2638  Because `dynamicController` will sync the watch events of CRD resource only after sending the watch request of R resource to `metaServer`, so I need to establish a watch connection when `edgecore` starts, so that the CRD created by the user can be sent to the edge and trigger me to update the crdMapper.